### PR TITLE
Delegate card

### PR DESCRIPTION
**Pull request by dnlklmn**

### CODE REVIEW

File: src/ui/layouts/Delegation.tsx [hunk 1]
```diff
@@ -121,7 +121,7 @@ export const DelegateSection = ({ delegates }: { delegates: Delegate[] }) => {
               />
             </div>
           </div>
-          <div className="flex flex-row flex-wrap items-center justify-start gap-y-4 gap-x-7">
+          <div className="grid grid-cols-3 flex-wrap items-center justify-start gap-6">
```
Comment (hamidra) [inline src/ui/layouts/Delegation.tsx:124]: Doesn't the grid layout need to be responsive?! currently it breaks the layout on small screens.

**Comment by jeluard** [comment]
@dnlklmn Are you rebasing your branch? It seems weird that this PR has so many commits.

File: src/ui/layouts/Delegation.tsx (hunk 1 above)
Comment (dnlklmn) [reply src/ui/layouts/Delegation.tsx:124]: ah didnt check on smaller screens yet, should be responsive yea, will fix it tomorrow

**Comment by dnlklmn** [comment]
> @dnlklmn Are you rebasing your branch? It seems weird that this PR has so many commits.

yea sorry no idea how that happened.. new to githubbing with others <3 

why is all the other commits included here that were merged with the previous pr?

**Comment by hamidra** [comment]
I've seen this issue before with Github. I believe their index gets messed up sometimes. I won't worry much about commit history as long as the file diffs match what is intended to be merged.

File: src/ui/layouts/Delegation.tsx (hunk 1 above)
Comment (dnlklmn) [reply src/ui/layouts/Delegation.tsx]: made some additional fixes to make it look decent on mobile. filters for delegates are not perfect, but i guess those will change anyway

**Pull request closed without merging**